src/main/scala/org/allenai/plugins/CoreDependencies.scala

```diff
@@ -95,6 +95,20 @@ trait CoreDependencies {
   // Spray json (separate from Spray toolkit)
   val sprayJson = "io.spray" %% "spray-json" % "1.3.1"
 
+  // Slick for database integration
+  // TODO consider upgrading to slick 3.0. All of our existing projects use
+  // slick 2.1.0, but we should considering migrating to 3.0.
```

Comment:
After talking with marks and looking here: http://slick.typesafe.com/doc/3.0.0/api/index.html#slick.jdbc.JdbcBackend$DatabaseDef it should be possible to recover{} from a Failure since a Future is returned from the run() method.